examples/3D_vectors_through_time.py

```diff
@@ -0,0 +1,112 @@
+"""
+3D vector field and image evolving through time
+================================================
+
+This example is from a mechanical test on a granular material, where multiple 3D volume have been acquired using x-ray tomography during axial loading.
+We have used [`spam`](https://www.spam-project.dev/) to label individual particles (with watershed and some tidying up manually), and these particles are tracked through time with 3D "discrete" volume correlation *incrementally*:
+t0 → t1 and then t1 → t2.
+Although we're also measuring rotations (and strains), but here we're interested to visualise the displacement field on top of the image to spot tracking errors.
```

Comment:
```suggestion
We're also measuring rotations and strains, but this example visualizes the displacement field on top of the image to spot tracking errors.
```
note I have no idea if we need to use the American vs British English spellings 🤷 sometimes the spell checker complains

Reply:
This is really up to you, but I don't know how to write in american English, so it won't be me changing it!